### ISSUE TITLE
I2C address ignored???

### ISSUE THREAD

**Issue by jasperza** [garmin/LIDARLite_Arduino_Library#9]
It seems as if the library ignores the i2c addess of the lidar lite, it returns readings no matter what address is used.

**Comment by JoeDurand3** [comment]
Yes - follow the instructions under the Operation Information section.

If you're having issues, after setting an address, try shifting your intended address << 1, as the least significant bit in that register does not count as part of the address.
